tsconfig.json

```diff
@@ -4,7 +4,8 @@
     "declaration": true,
     "declarationMap": true,
     "noImplicitAny": true,
-    "outDir": "./dist"
+    "outDir": "./dist",
+    "resolveJsonModule": true
```

Comment:
We played this option when we were trying to read the package version as part of the [getUserAgent](https://github.com/sigstore/sigstore-js/blob/main/src/util/ua.ts) function. What we observed is that it kinda creates a mess when the package is tar'ed (`npm pack`). Ultimately, we just resorted to [requiring](https://github.com/sigstore/sigstore-js/blob/main/src/util/ua.ts#L22) the package JSON so that we didn't have to use this setting at all.

Reply:
Ah I see, I'll change this to user `require`. Thanks